src/torchcodec/decoders/_video_decoder.py

```diff
@@ -41,6 +41,8 @@ class VideoDecoder:
             instances of ``VideoDecoder`` in parallel. Use a higher number for multi-threaded
             decoding which is best if you are running a single instance of ``VideoDecoder``.
             Default: 1.
+        device (str or torch.device, optional): The device to use for decoding.
+
```

Comment:
This wasn't introduced in this PR, but we might as well fix it here: the `.. note::` below should be part of the parameter description of the `dimension_order` parameter. Do you mind moving it back up?

Reply:
done